src/test/java/net/sf/jabref/logic/cleanup/CleanupWorkerTest.java

```diff
@@ -307,11 +308,11 @@ public void cleanupLatexMergesTwoLatexMathEnvironments() {
     public void convertToBiblatexMovesJournalToJournalTitle() {
         CleanupPreset preset = new CleanupPreset(CleanupPreset.CleanupStep.CONVERT_TO_BIBLATEX);
         BibEntry entry = new BibEntry();
-        entry.setField("journal", "test");
+        entry.setField(FieldName.JOURNAL, "test");
```

Comment:
Please revert these changes, as hard-coded strings in tests are actually desirable. The logic for the conversation moves FieldName.JOURNAL to FieldName,JOURNALTITLE. Hence if somebody changes `FieldName.JOURNAL = "stupid"` the test still passes but the functionality is broken.